### ISSUE TITLE
[Question]: uie 实体抽取为什会在windows和linux环境下会差异很大

### ISSUE THREAD

**Issue by xiu86** [PaddlePaddle/PaddleNLP#5476]
### 请提出你的问题

同样的训练数据，同样的命令，在windows和linux下的评估结果差异很大，为什么会这样 #
linux
<img width="1363" alt="WechatIMG821" src="https://user-images.githubusercontent.com/3351443/228764083-eee95c04-0eb6-4493-ad2e-fac4cb5da52b.png">
windows
![WechatIMG822](https://user-images.githubusercontent.com/3351443/228764084-be04d70b-d60b-48ab-bcb2-fa3f653ad06e.png)

**Comment by linjieccc** [comment]
@xiu86 训练集有多大呢，方便的话可以贴下微调的执行方式

**Comment by xiu86** [comment]
> @xiu86 训练集有多大呢，方便的话可以贴下微调的执行方式
训练数据20条左右
微调命令
`python3.9 finetune.py --device gpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 42 --model_name_or_path uie-base \
    --output_dir $finetuned_model --train_path data/pub_zb_contact/train.txt --dev_path data/pub_zb_contact/dev.txt --max_seq_length 512  \
    --per_device_eval_batch_size 16 --per_device_train_batch_size  16 --num_train_epochs 20 --learning_rate 1e-5 \
    --label_names 'start_positions' 'end_positions' --do_train --do_eval --do_export --export_model_dir $finetuned_model --overwrite_output_dir \
    --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end  True --save_total_limit 1`
